src/EnergyPlus/DataHeatBalance.cc

```diff
@@ -1294,71 +1294,64 @@ Real64 ComputeNominalUwithConvCoeffs(EnergyPlusData &state,
     //      Interior horizontal surfaces, heat flow up     IP: 0.61  SI: 0.1074
     //      Interior horizontal surfaces, heat flow down   IP: 0.92  SI: 0.1620
     //      Interior vertical surfaces                     IP: 0.68  SI: 0.1198
-    // This section shows the same value in 90.1-2007 and 90.2-2010
-
-    // Using/Aliasing
-    using DataSurfaces::ExternalEnvironment;
-    using DataSurfaces::Ground;
-    using DataSurfaces::GroundFCfactorMethod;
-    using DataSurfaces::SurfaceClass;
+    // This section shows the same value in 90.1-2010 and 90.2-2010
+    // Note that this report does not use the semi-exterior surface value because
+    // EnergyPlus does not have a way to specifically tell whether or not a surface
+    // is connected to a semi-exterior area of the building.  Users can always use
+    // the Nominal U-Value to manually calculated this.  The values calculated here
+    // are simply reported to the EIO file and not used for any calculations.
 
     // Return value
     Real64 NominalUwithConvCoeffs; // return value
 
+    static constexpr std::array<Real64, static_cast<int>(DataSurfaces::SurfaceClass::Num)> filmCoefs = {
+        0.0,       // None
+        0.1197548, // Wall
+        0.1620212, // Floor
+        0.1074271, // Roof
+        0.0,       // IntMass
+        0.0,       // Detached_B
+        0.0,       // Detached_F
+        0.1197548, // Window
+        0.1197548, // GlassDoor
+        0.1197548, // Door
+        0.0,       // Shading
+        0.0,       // Overhang
+        0.0,       // Fin
+        0.0,       // TDD_Dome
+        0.0        // TDD_Diffuser
+    };             // If anything added to the enum SurfaceClass, adjust this list appropriately
```

Comment:
This is fine, and as I mentioned on the call, if the _size_ of the SurfaceClass enum changes, the compiler should warn/error that this initializer list is the wrong size.  Now, if someone reorders the enums or something, then well, we'll have to rely on your comment there.  But that sort of thing goes for a lot of of stuff in E+.  This is good.